v2/fleet/mesos-slave.service

```diff
@@ -33,7 +31,6 @@ ExecStart=/usr/bin/sh -c "sudo /usr/bin/docker run \
     --containerizers=docker,mesos \
     --executor_registration_timeout=10mins \
     --hostname=`curl -s http://169.254.169.254/latest/meta-data/public-hostname` \
-    --isolation=cgroups/cpu,cgroups/mem \
```

Comment:
I don't mind either way, just something to keep in mind since we're aiming for more stability